Gemfile

```diff
@@ -39,10 +40,4 @@ group :development do
   gem 'web-console', '>= 3.3.0'
 end
 
-group :test do
```

Comment:
why are you removing these gems?

Reply:
There were duplicated after rebase from remote develop branch.
Error looked liked this:
"Errors on lines you didn't modify:
/home/mayar/study/Ruby_Combats/Gemfile:44:3: C: Bundler/DuplicatedGem: Gem `database_cleaner` requirements already given on line 28 of the Gemfile.
/home/mayar/study/Ruby_Combats/Gemfile:45:3: C: Bundler/DuplicatedGem: Gem `faker` requirements already given on line 29 of the Gemfile.
/home/mayar/study/Ruby_Combats/Gemfile:46:3: C: Bundler/DuplicatedGem: Gem `shoulda-matchers` requirements already given on line 30 of the Gemfile."

Reply:
if you remove them then you will not have them. I do not see duplicates of these gems inside the Gemfile

Reply:
line 27-31, duplicate Test gems twice? than in line 42...

Reply:
I do not see these

gem 'database_cleaner', '~> 1.5'
gem 'faker', '~> 1.6.1'
gem 'shoulda-matchers', '~> 3.0', require: false

to be duplicated.

Reply:
These gems were duplicated on my local branch after rebasing from remote branch. And because of that I didn't pass precommit test. So I decided to delete duplicated part of code to fix that error.
Can you advise me better solution to fix that problem?

Reply:
Remote <develop> branch in github has duplicated

Reply:
![screenshot from 2017-09-28 22-54-46](https://user-images.githubusercontent.com/14543503/30987811-f9bcc56e-a4a0-11e7-8f27-abc88899c7f0.png)
line 27 and line 42

Reply:
I see. Who did this? How did it pass the code review?

Reply:
I add extra line when resolve conflict :(

Reply:
@vlodkopits you add them back too in the PR https://github.com/rokor/Ruby_Combats/pull/4